### ISSUE TITLE
Unknown fuction: domain_exists on version 2.0.1

### ISSUE THREAD

**Issue by eloyvaldera-te** [biemond/biemond-orawls#413]
Hi!

I have found this issue after update to version 2.0.1 :

```
Error: Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Unknown function: 'domain_exists'. at /etc/puppetlabs/code/environments/production/modules/orawls/manifests/copydomain.pp:93:12  at /etc/puppetlabs/code/environments/production/modules/profile/manifests/orawls/wls/managed_server.pp:233 on node managed-server.t0.inet
```

**Comment by biemond** [comment]
You are right , I forget about this puppet function . I will fix it tomorrow

In the meantime you can use version 2.0.0, not much changed

**Comment by biemond** [comment]
fixed by 2.0.2

thanks again
